pom.xml

```diff
@@ -50,7 +50,7 @@
     </scm>
 
     <properties>
-        <confluent.version>3.3.0-SNAPSHOT</confluent.version>
+        <confluent.version>${project.version}</confluent.version>
```

Comment:
@aayars This seems like a good improvement (we should avoid duplicating the version number more places than necessary), but figure it's worth checking that this won't break the version bumping tooling.